kasa/cli.py

```diff
@@ -81,6 +81,36 @@ def cli(ctx, host, alias, target, debug, bulb, plug, strip):
         ctx.invoke(state)
 
 
+@cli.group()
+@pass_dev
+def wifi(dev):
+    """Commands to control wifi settings."""
+    pass
+
+
+@wifi.command()
+@pass_dev
+def scan(dev):
+    """Scan for available wifi networks."""
+    click.echo("Scanning for wifi networks, wait a second..")
+    devs = asyncio.run(dev.wifi_scan())
+    click.echo("Found %s wifi networks!" % len(devs))
+    for dev in devs:
+        click.echo("\t %s" % dev)
```

Comment:
Why no f-strings?

Reply:
Good catch! I converted most of the non-f-string uses to f-strings.